src/cascadia/TerminalApp/TerminalPage.idl

```diff
@@ -97,6 +97,7 @@ namespace TerminalApp
 
         event Windows.Foundation.TypedEventHandler<Object, Object> OpenSystemMenu;
         event Windows.Foundation.TypedEventHandler<Object, Microsoft.Terminal.Control.ShowWindowArgs> ShowWindowChanged;
+        event Windows.Foundation.TypedEventHandler<Object, Windows.Foundation.Collections.IVectorView<Microsoft.Terminal.Settings.Model.SettingsLoadWarnings> > ShowLoadWarningsDialog;
```

Comment:
Fun fact: you need the space here - `SettingsLoadWarnings> > ShowLoadWarningsDialog` or else the MIDL compiler gets angry and confused

Reply:
yup! C++03 template rules!